content/docs/dvclive/get-started.md

```diff
@@ -116,51 +78,98 @@ for epoch in range(NUM_EPOCHS):
     live.next_step()
 ```
 
-## Outputs
+</tab>
+</toggle>
+
+Check the [ML Frameworks](/doc/dvclive/api-reference/ml-frameworks) page for
+more details and other supported frameworks.
+
+If you prefer to use DVCLive directly, check the overview of the
+[Python API](/doc/dvclive/api-reference).
 
-After you run your training code, you should see the following content in the
-project:
+### Run the script
+
+Once you have added DVCLive to our python code, you can run the script as you
+would usually do:
 
 ```dvc
-$ tree
-├── dvclive
-│   ├── images
-│   │   └── img.png
-│   ├── params.yaml
-│   ├── plots
-│   │   └── roc.json
-│   └── scalars
-│       └── acc.tsv
-└── dvclive.json
+$ python train.py
+```
+
+DVCLive will generate a [report](/doc/dvclive/outputs#report) in
+`dvclive/report.html` containing all the logged data. It will be automatically
+updated during training on each `step` update:
+
+![HTML report](/img/dvclive-html.gif).
+
+You can find more details about the structure of files generated by DVCLive in
+the [Output folder structure](/doc/dvclive/outputs).
```

Comment:
It might be worth showing some example output here, like `dvclive.json` and `dvclive/scalars`, especially since you mention them below without any prior context on this page.